src/default_parameters.jl

```diff
@@ -20,7 +20,7 @@
     R::Real=6.371e6                     # Earth's radius [m]
 
     # SCALE
-    scale::Real=2^6                     # multiplicative scale for the momentum equations u,v
+    scale::Int=2^6                      # multiplicative scale for the momentum equations u,v
```

Comment:
I see that ideally you'd want scaling to be a power of two which is always an int, but it still works fine with `Float64` so let's not be unnecessarily restrictive?